### DISCUSSION TITLE
Does the number of the NumGrid_Z and NumGrid_Y influence the motion responce?

### DISCUSSION THREAD

**Discussion by haoshuning** [OpenFAST/openfast#2318] [Q&A]
Dear @jjonkman ,
 I'm trying to simulate 3h API wind spectrum random winds with turbsim，but when I adjust the UsableTime to the 10800，the  program rasises the following error,just as the picture shows. So I change the NumGrid_Y and Number_Z to 3, later the wind  can be simulated successfully. However, I am not sure if this will affect the calculations for turbine operation and platform motion in the next step.  
 My second question is that when I choose the API model and the WindProfileType is also the API, the output wind speed of the hub height is slightly small, I calculate the height coefficient is just equel to 0.07. Does the situation is normal , and can I use this wind file with low wind speeds for subsequent calculations?
I am really  looking forward to your reply. Thank you sincerely!!!! 
![11111111](https://github.com/user-attachments/assets/8743d8f8-b16f-4dec-9be6-b5b320a04858)

**Comment by jjonkman** [comment]
Dear @haoshuning,

Cholesky factorization errors are a known issue of TurbSim when applying the API model when the spatial discretization is too fine, as reported in the following OpenFAST issue: https://github.com/OpenFAST/openfast/issues/386.  Reducing `NumGrid_Y` and `NumGrid_Z` to 3 is not a valid solution given that the resulting grid would be very coarse and our recommended discretization size is max chord.  Can you increase `NumGrid_Y` and `NumGrid_Z` to a more reasonable number without triggering the error?

Best regards,
